shared/anachronism_detector.py

```
@@ -0,0 +1,355 @@
+        """Print a formatted report of scan results."""
+        print("\n" + "=" * 60)
+        print("ANACHRONISM DETECTION REPORT")
+        print("=" * 60)
+
+        print(f"\nFiles scanned: {results['total_files']}")
+        print(f"Total samples: {results['total_samples']:,}")
+        print(f"Files with issues: {results['files_with_issues']}")
+        print(f"Samples with issues: {results['samples_with_issues']}")
+        print(f"Total issues found: {results['total_issues']}")
+
+        print(f"\nIssues by type:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)